### DISCUSSION TITLE
KEDA Chart v2.14.0

### DISCUSSION THREAD

**Discussion by JorTurFer** [kedacore/charts#631] [Announcements]
Chart: v2.14.0 | App: v2.14.0

## TL;DR

```shell
helm repo add kedacore https://kedacore.github.io/charts
helm repo update
helm install keda kedacore/keda --version 2.14.0 -n keda --create-namespace
```

### Features

- Support for KEDA v2.14.0 ([release notes](https://github.com/kedacore/keda/releases/tag/v2.14.0))
- Improved RBAC

## New Contributors
* @lpetrazickisupgrade made their first contribution in https://github.com/kedacore/charts/pull/608
* @iamKunal made their first contribution in https://github.com/kedacore/charts/pull/617
* @roee-hersh made their first contribution in https://github.com/kedacore/charts/pull/618
* @jkremser made their first contribution in https://github.com/kedacore/charts/pull/619

**Full Changelog**: https://github.com/kedacore/charts/compare/v2.13.2...v2.14.0

<hr /><em>This discussion was created from the release <a href='https://github.com/kedacore/charts/releases/tag/v2.14.0'>KEDA Chart v2.14.0</a>.</em>
